src/main/java/com/bugsnag/android/ExceptionHandler.java

```diff
@@ -47,18 +51,30 @@ public ExceptionHandler(UncaughtExceptionHandler originalHandler) {
         this.originalHandler = originalHandler;
     }
 
+    @Override
     public void uncaughtException(@NonNull Thread t, @NonNull Throwable e) {
+        boolean strictModeThrowable = strictModeHandler.isStrictModeThrowable(e);
+      
         // Notify any subscribed clients of the uncaught exception
-        for(Client client : clientMap.keySet()) {
-            client.cacheAndNotify(e, Severity.ERROR);
+        for (Client client : clientMap.keySet()) {
+
+            if (strictModeThrowable) { // add strictmode policy violation to metadata
+                String violationDesc = strictModeHandler.getViolationDescription(e.getMessage());
+                MetaData metaData = new MetaData(
+                    Collections.<String, Object>singletonMap(KEY_STRICT_MODE_VIOLATION, violationDesc));
+                client.cacheAndNotify(e, Severity.ERROR, metaData);
+            }
+
+            client.cacheAndNotify(e, Severity.ERROR, null);
```

Comment:
I noticed that reports are being sent in duplicate, one with and one without the strictmode metadata. I assume there should be an `else` clause wrapping this line.